src/openforms/emails/digest.py

```diff
@@ -257,3 +297,154 @@ def collect_invalid_certificates() -> list[InvalidCertificate]:
+        # so we show the exact ones
+        plugin_cls = type(plugin)
+        if plugin_cls not in checked_plugins:
+            try:
+                plugin.check_config()
+            # we always raise an InvalidPluginConfiguration exception even when we have
+            # errors like HTTPError
+            except InvalidPluginConfiguration as e:
+                invalid_registration_backends.append(
+                    InvalidRegistrationBackend(
+                        config_name=plugin.verbose_name,
+                        exception_message=e.args[0],
+                        form_name=form_name,
+                        form_id=form.id,
+                    )
+                )
+            checked_plugins.add(plugin_cls)
+
+        # errors in the form level can be more detailed so here we want to show
+        # a more general error and the link to the broken form to investigate
+        serializer = plugin.configuration_options(
+            data=backend_options,
+            context={"validate_business_logic": True},
+        )
+
+        try:
+            serializer.is_valid(raise_exception=True)
+        except serializers.ValidationError:
+            invalid_registration_backends.append(
+                InvalidRegistrationBackend(
+                    config_name=plugin.verbose_name,
+                    exception_message=_(
+                        "Invalid registration backend configuration detected"
+                    ),
+                    form_name=form_name,
+                    form_id=form.id,
+                )
+            )
+
+    return invalid_registration_backends
+
+
+def collect_invalid_logic_rules() -> list[InvalidLogicRule]:
+    forms = Form.objects.live().iterator()
+    static_variables = {
+        var.key: {"source": var.source, "type": var.data_type}
+        for var in get_static_variables()
+    }
+
+    invalid_logic_rules = []
+    for form in forms:
+        form_variables = {
+            form_variable.key: {
+                "source": form_variable.source,
+                "type": form_variable.data_type,
+            }
+            for form_variable in form.formvariable_set.all()
+        }
+
+        all_keys = list(static_variables) + list(form_variables)
+
+        form_logics = FormLogic.objects.filter(form=form)
+
+        form_logics_vars = []
+        for logic in form_logics:
+            for var in introspect_json_logic(logic.json_logic_trigger).get_input_keys():
+                form_logics_vars.append(var)
+            for action in logic.actions:
+                if action["action"]["type"] == LogicActionTypes.variable:
+                    form_logics_vars.append(InputVar(key=action["variable"]))
+                    expression = action["action"]["value"]
+                    form_logics_vars += [
+                        var
+                        for var in introspect_json_logic(expression).get_input_keys()
+                    ]
```

Comment:
(no need to fix this, just for the future)

the list comprehension isn't necessary, `get_inputs_keys()` already returns a list of input vars:

```suggestion
                    form_logics_vars += introspect_json_logic(expression).get_input_keys()
```